pkg/systemd/host.css

```diff
@@ -283,6 +283,15 @@ body {
     word-wrap: break-word;
 }
 
+.system-information-updates > .fa,
+.system-information-updates > .spinner {
+    line-height: inherit;
+}
+.system-information-updates > .spinner {
+    /* Offset for alignment: 2px = 16px - 12px height - 1px border */
+    margin: 3px 0 0;
```

Comment:
I hate magic numbers. I could align to center, but if the text string wraps, then it's aligned to center and not baseline. Perhaps that would be better. Perhaps having it aligned to baseline (like it is) and having the offset (with a comment) is better.

Reply:
Had a typo where the comment was wrong. Fixed and force-pushed. :roll_eyes: